data/json/items/generic.json

```diff
@@ -2979,6 +2979,21 @@
     "use_action": "DISASSEMBLE",
     "flags": [ "NO_REPAIR" ]
   },
+  {
+    "id": "premium_survival_kit",
+    "type": "GENERIC",
+    "category": "tools",
+    "name": "survival kit",
+    "description": "The ultimate survival kit purchased either by the wealthy or the dedicated urban survivalist.  It contains tools and items to help you survive in case of an emergency.  Disassemble to get its content.",
```

Comment:
```suggestion
    "description": "The ultimate survival kit purchased either by the wealthy or the dedicated urban survivalist.  It contains tools and items to help you survive in case of an emergency.  Disassemble to get its contents.",
```